ui/panels/render_properties.py

```diff
@@ -0,0 +1,49 @@
+import bpy
+from .dream_texture import create_panel, prompt_panel, advanced_panel
+
+class RenderPropertiesPanel(bpy.types.Panel):
+    """Panel for Dream Textures render properties"""
+    bl_label = "Dream Textures"
+    bl_idname = "DREAM_PT_dream_render_properties_panel"
+    bl_space_type = 'PROPERTIES'
+    bl_region_type = 'WINDOW'
+    bl_context = 'render'
+    bl_options = {'DEFAULT_CLOSED'}
+
+    @classmethod
+    def poll(self, context):
+        return context.scene.render.engine == 'CYCLES'
+
+    def draw_header(self, context):
+        self.layout.prop(context.scene, "dream_textures_render_properties_enabled", text="")
+
+    def draw(self, context):
+        layout = self.layout
+        layout.use_property_split = True
+        layout.use_property_decorate = False
+        layout.active = context.scene.dream_textures_render_properties_enabled
+
+        layout.prop(context.scene.dream_textures_render_properties_prompt, "strength")
+
+def render_properties_panels():
+    yield RenderPropertiesPanel
+    def get_prompt(context):
+        return context.scene.dream_textures_render_properties_prompt
+    space_type = RenderPropertiesPanel.bl_space_type
+    region_type = RenderPropertiesPanel.bl_region_type
+    panels = [
```

Comment:
May want to add a size panel so renders can be done in higher resolutions.

Reply:
Or potentially use the output properties resolution instead.

Reply:
Using the output properties resolution would also resolve the issue from #245, correct? Although it would still need to require a multiple of 64, so perhaps that could simply show an error right away if detected, asking the user to update their output properties.